Docker/debian/debian-10-vars.json

```diff
@@ -0,0 +1,14 @@
+{
+  "_comment": "Build with `packer build -var-file=debian-10-vars.json debian-10.json`",
+  "docker_image": "saltstack/ci-debian-10:base",
+  "docker_repository": "saltstack",
+  "docker_image_prefix": "ci",
+  "build_type": "ci",
+  "os_name": "Debian",
+  "os_version": "10",
+  "bootstrap_args": "-qXU -x python2",
```

Comment:
should this be python3 ?

Reply:
No? https://github.com/saltstack/os-imager/blob/ci/AWS/debian/debian-10-us-west-2.json

Reply:
we should change them both, but we can do that later.